CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ internal API changes are not present.
 Main (unreleased)
 -----------------
 
+### Features
+
+- (beta) Add app o11y integration. This depends on integrations-next being enabled
```

Comment:
I think given we're already thinking about a v2 that goes all in on OTLP payloads, maybe this should be experimental instead of beta?

@mattdurham do you think this should have a feature flag?